kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/ConnectProducer.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
I believe we had to use this class in the [past](https://github.com/confluentinc/kafka-connect-gcp-spanner/blob/master/src/test/java/io/confluent/connect/gcp/spanner/integration/ConnectProducer.java) because we are producing SinkRecords. I like the approach you suggested and tried it, but turns out that `formatter.formatValue` only accepts SinkRecords, not source records as we need for this sink. Let me know if you have any other ideas.

Reply:
why do you need to produce SourceRecords... SourceRecords are for SourceConnectors. I have been using this approach in all Sink connectors and it has been working fine

Reply:
Awesome, that worked. I was going off of a previous example...